projects/kit/components/input-files/input-files.directive.ts

```diff
@@ -0,0 +1,65 @@
+import {Directive, ElementRef, HostBinding, HostListener, Inject} from '@angular/core';
+import {TuiIdService} from '@taiga-ui/cdk';
+
+// eslint-disable-next-line import/no-cycle
+import {TuiInputFilesComponent} from './input-files.component';
+import {TUI_INPUT_FILES_OPTIONS, TuiInputFilesOptions} from './input-files.options';
+
+@Directive({
+    selector: `input[tuiInputFiles]`,
+    host: {
+        type: `file`,
+        class: `t-native`,
+        '(mousedown.prevent.silent)': `(0)`,
```

Comment:
We can move those events in the parent container inside `InputFilesComponent` because they both bubble.

Reply:
fixed